composer.json

```diff
@@ -22,7 +22,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^7.",
-        "clearfacts/cf-codestyle": "^1.0"
+        "clearfacts/cf-codestyle": "^1.0",
+        "symfony/messenger": "^5.4"
```

Comment:
das eerder iets voor in de suggest ipv de require dev ni?

Reply:
during dev nodig omdat ge die serializer extend